### ISSUE TITLE
[bug]: [gh-r] Error: No GitHub release assets found for v7.3.2

### ISSUE THREAD

**Issue by hrai** [zdharma-continuum/zinit#476]
### What happened?

I have this code in my .zshrc

```
    zinit ice from"gh-r" as"program" bpick"*.deb"
    zinit light powershell/powershell
```
I get the error when I know there are release assets.

```
[gh-r] Error: bpick ice found no release assets. To fix, modify the bpick glob pattern *.deb

[gh-r] Error: No GitHub release assets found for v7.3.2
```

### Steps to reproduce

    zinit ice from"gh-r" as"program" bpick"*.deb"
    zinit light powershell/powershell

### Screenshots and recordings

![image](https://user-images.githubusercontent.com/4055444/218893001-59e9f783-5bf1-4061-bf49-8b2bbd7948aa.png)

### Operating System & Version

OS: linux-gnu | Vendor: ubuntu | Machine: x86_64 | CPU: x86_64 | Processor: x86_64 | Hardware: x86_64

### Zsh version

zsh 5.9 (x86_64-ubuntu-linux-gnu)

### Terminal emulator

xterm-256color

### If using WSL on Windows, which version of WSL

WSL 2

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vladdoster** [comment]
@hrai thanks for reporting this.

looking into this.

**Comment by vladdoster** [comment]
@hrai,

The repository is case-sensitive.

> https://github.com/PowerShell/PowerShell

```zsh
zinit for \
    from'gh-r' \
    lbin'!**/pwsh->pwsh' \
    bpick'*linux-x64*' \
    null \
  @PowerShell/PowerShell
```

<img width="1310" alt="Screenshot 2023-02-16 at 03 10 31" src="https://user-images.githubusercontent.com/10052309/219320017-18427334-ce26-469d-b898-5ee43060a284.png">

**Comment by vladdoster** [comment]
The `bpick` is only needed due to various Linux assets. Additionally, we filter out Linux packages (i.e., deb, rpm).

**Comment by hrai** [comment]
@vladdoster  Currently it's sourcing everything in the folder, is there a way to source only the correct executable?

**Comment by vladdoster** [comment]
What do you mean sourcing? Please provide a screenshot.

**Comment by hrai** [comment]
@vladdoster it's this block from your screenshot.

**Comment by vladdoster** [comment]
@hrai, 

Circling back on this issue.

Addressing the concern regarding Zinit sourcing a ton of miscellaneous executables.

I think this is a non-issue. If you are referring to the log output that states Zinit has marked 300+ files as executable, it a a non-issue. If you look at:

```
la -l $ZPFX
```

You will see that only those files are ones that get added to your `$path`.

Additionally, if you have no more issues  could we consider the issue resolved and close the issue?

Can this issue be closed?

**Comment by vladdoster** [comment]
@hrai 

> @vladdoster it's this block from your screenshot.

Which line exactly?

**Comment by vladdoster** [comment]
@hrai,

Fixed in #479.

**Comment by hrai** [comment]
Thanks @vladdoster
